README.md

```diff
@@ -40,12 +41,18 @@ To test the setup action set the required environment variables and execute `set
 ```bash
 $Env:RESOURCE_GROUP_OVERRIDE=yourResourceGroup
 $Env:REGION_OVERRIDE=yourRegion
-.\setup.ps1 -ASBName psw-asb-1 -ConnectionStringName AzureServiceBus_ConnectionString -Tag setup-azureservicebus-action
+# Replace the principal ID with the appropriate principal ID that you used to log into AZ CLI
+$azureCredentials = @"
+{
+     "principalId": "a28b36b8-2243-494e-9028-0e94df179913",
```

Comment:
is this meant to be hard coded?

Reply:
The ID is just a sample. That's why there is a comment above that line